packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerConstants.kt

```diff
@@ -35,8 +35,6 @@ object ImagePickerConstants {
   const val OPTION_EXIF = "exif"
   const val OPTION_VIDEO_MAX_DURATION = "videoMaxDuration"
 
-  // We need to explicitly get latitude, longitude, altitude with their specific accessor functions
-  // separately so we skip them in this list.
```

Comment:
Before this change we needed constants only to copy exif data to `exif` result parameter. We didn't need `lat*, long*, alt*` because we get this [in this way](https://github.com/expo/expo/blob/2607ac767410d2ae72dabe53d682245024bbd3f1/packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt#L122)  Now I use it also to copy exif from original to edited image. I could add this manually but I believe that this solution is more elegant. During copying exif data to `exif` parameter we overwrite it so I think it's not a problem.